### PR TITLE
Updated rkt demo and docs

### PR DESCRIPTION
**Pull request by gunjan5**

### CODE REVIEW

**Comment by djosborne** [comment]
@gunjan5 can you modify this so there's no file location movement yet? The diff is difficult to make sense of atm. Can do a follow up PR to toy with structure.

**Comment by gunjan5** [comment]
@djosborne there's more than location movement. Location movement still shows up as normal diff, it's just the file I modified and added some more content and then broke it down to two files `/vagrant-coreos/index` and `/tutorials/basic`. I'm creating a branch with no location change

**Comment by gunjan5** [comment]
@djosborne toy PR to see the diff view https://github.com/gunjan5/calico/pull/1/files

**Comment by djosborne** [comment]
I believe this Fixes #26 and Fixes #229

File: master/getting-started/rkt/tutorials/basic.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+--volume=birdctl,kind=host,source=/var/run/calico,readOnly=false \
+--mount volume=birdctl,target=/var/run/calico \
+--volume=mods,kind=host,source=/lib/modules,readOnly=false  \
+--mount volume=mods,target=/lib/modules \
+--volume=logs,kind=host,source=/var/log/calico,readOnly=false \
+--mount volume=logs,target=/var/log/calico \
+--set-env=IP=autodetect --net=host quay.io/calico/node:latest &
+```
+
+This will create a rkt container called `calico-node`.
```
Comment (robbrockbank) [inline master/getting-started/rkt/tutorials/basic.md]: Is it really called calico-node?

Comment (gunjan5) [inline master/getting-started/rkt/tutorials/basic.md]: hmm not really, I'll change it

File: _data/master/navbars/getting-started.yml [hunk 2]
```diff
@@ -145,10 +145,12 @@ toc:
     path: /getting-started/rkt/
   - title: Installation
     section:
-    - title: Vagrant
-      path: /getting-started/rkt/vagrant/
+    - title: 'Vagrant/VirtualBox: CoreOS'
```
Comment (caseydavenport) [inline _data/master/navbars/getting-started.yml]: Why the single quotes? I think we don't want that.

File: master/getting-started/rkt/installation/cloud-config/other-node-config.yaml [hunk 3]
```diff
@@ -2,9 +2,9 @@
 ---
 coreos:
   update:
-    reboot-strategy: off
+    reboot-strategy: 'off'
```
Comment (caseydavenport) [inline master/getting-started/rkt/installation/cloud-config/other-node-config.yaml]: I don't think this is necessary, is it?

File: _data/master/navbars/getting-started.yml (hunk 2 above)
Comment (robbrockbank) [inline _data/master/navbars/getting-started.yml]: That might be required given the string has a colon in it...

Comment (robbrockbank) [inline _data/master/navbars/getting-started.yml]: Could just change to Vagrant/VirtualBox/CoreOS    :-)

Comment (robbrockbank) [inline _data/master/navbars/getting-started.yml]: without the smiley face.

**Comment by robbrockbank** [comment]
This was checked in under another PR... Closing.

**Pull request closed without merging**